### DISCUSSION TITLE
Help memory leak?

### DISCUSSION THREAD

**Discussion by moreje** [FlareSolverr/FlareSolverr#398] [General]
Hello all,
my flaresolverr docker is putting my NAS down!
I migrated all my dockers from an old system to a new Ryzen 5 / 16G RAM PC
Flaresolverr runs into a docker along with all servarr dockers (prowlarr, sonarr, etc...)
My issue is that after few hours, flaresolverr is putting my PC out of ressources and irresponsive. 
looks like many firefox processes are eating my RAM & CPU
My OS is based on Debian 11, OMV 6 release (so no graphical environnment)
any ideas?
Thank you
![Capture d’écran du 2022-05-25 18-59-08](https://user-images.githubusercontent.com/5374164/170322235-79c20f92-5295-43be-89d6-bf7f31f63424.png)

**Comment by ngosang** [comment]
FlareSolverr is stable in my environment but other users reported the same issue as you.
Make sure you have installed the latest version and set some limits in the container to limit the damage => https://www.baeldung.com/ops/docker-memory-limit
